### ISSUE TITLE
Fix errror: "ModuleNotFoundError: No module named 'telegram.client'"

### ISSUE THREAD

**Issue by nlhomme** [nlhomme/pytgvoip#6]
```
root@proliant:[~/git/pytgvoip]|19:51:50|# docker run -it nlhomme/pytgvoip:latest
Traceback (most recent call last):
  File "/root/tgcall.py", line 10, in <module>
    from telegram.client import Telegram
ModuleNotFoundError: No module named 'telegram.client'
```

**Comment by Baptiste-Leterrier** [comment]
Got error:

root@7c6cae294119:~# python3 tgcall.py 
```
Traceback (most recent call last):
  File "tgcall.py", line 10, in <module>
    from telegram.client import Telegram
  File "/usr/local/lib/python3.7/dist-packages/telegram/client.py", line 16, in <module>
    from telegram.utils import AsyncResult
ImportError: cannot import name 'AsyncResult' from 'telegram.utils' (/usr/local/lib/python3.7/dist-packages/telegram/utils/__init__.py)
```

**Comment by nlhomme** [comment]
No luck with commit 919fb57479bcd9e31fea30862dcbeaa3b4995b71
```
Collecting python-telegram==11.1.0 (from -r requirements.txt (line 2))
[91m Could not find a version that satisfies the requirement python-telegram==11.1.0 (from -r requirements.txt (line 2)) (from versions: 0.2.0, 0.3.0, 0.3.1, 0.4.0, 0.5.0, 0.6.0, 0.6.1, 0.7.0, 0.8.0, 0.9.0, 0.10.0, 0.11.0, 0.12.0)
[0m
[91mNo matching distribution found for python-telegram==11.1.0 (from -r requirements.txt (line 2))
[0m
Removing intermediate container 20d392d09fc8
The command '/bin/sh -c pip3 install -r requirements.txt' returned a non-zero code: 1
```

**Comment by nlhomme** [comment]
Fixed with commits b0b0969 and 2a6346b
